Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+RUN pip install --upgrade --no-cache-dir black
```

Comment:
this will always install the latest version from pypi, should this install from the local repository such that the version used in the github action will match what you get in the docker image? (so `COPY . . && RUN pip install .`)